frontend/components/Dashboard/MyFeed/MyFeed.tsx

```diff
@@ -1,10 +1,11 @@
+// @ts-nocheck
 import React from 'react'
 
 import PostCard from '../PostCard'
 
 // import { Post } from '../../../generated/graphql' // TODO (robin-macpherson): See Post[] type todo below
 
-interface Props {
+type Props = {
```

Comment:
@robin-macpherson I wanted to get your feedback on this one before I made a lot of updates. After some research, I'm thinking that types for props as opposed to interfaces is better for React components. A very popular [React TypeScript cheatsheet](https://github.com/typescript-cheatsheets/react-typescript-cheatsheet#types-or-interfaces) with 12.5k github stars recommends types for props and state because its much better for unions between types, and in my other React projects where we're using interfaces, I've found them to be a bit harder to work with. Let me know whatcha think 😊

Reply:
Here's another good [article](https://medium.com/@martin_hotell/interface-vs-type-alias-in-typescript-2-7-2a8f1777af4c) on types vs interfaces

Reply:
Thanks for the cheatsheet and article, @nlitwin ! This makes sense and I'm fine with moving forward with this approach 😊💪🏼